pkg/ec2helper/ec2helper.go

```diff
@@ -1236,6 +1266,23 @@ func ValidateImageId(h *EC2Helper, imageId string) bool {
 	return image != nil
 }
 
+// Validate a filepath. Used as a function interface to validate question input
+func ValidateFilepath(h *EC2Helper, userFilePath string) bool {
```

Comment:
Doesn't `ValidateLaunchFlags` handle validating filepath and tags?

Reply:
Yes, but `ValidateLaunchFlags` is for _non-interactive mode_ **only**

For _interactive mode_ this function gets passed as a "validator" to check the user's input.

The refactoring that needs to be done here is related to the other comment about skipping error check on file open. I'd like to get the functionality out there and address these refactors in a separate PR once I get some solid e2e tests in place-- does that work?

Reply:
Sure. Sounds good :)